pillar/environment-preview-public.sls

```diff
@@ -3,5 +3,4 @@ elife:
 
 journal:
     feature_xpub: false
-    submit_url: http://submit.elifesciences.org/
-    #submit_url: https://xpub.elifesciences.org/login
+    submit_url: https://xpub.elifesciences.org/login
```

Comment:
Probably ok. Means that the submit button goes straight to xPub, but the environment is internal.